modules/UI/videolayout/VideoLayout.js

```diff
@@ -331,14 +331,11 @@ var VideoLayout = {
 
         remoteVideo.addRemoteStreamElement(stream);
 
-        // if track is muted make sure we reflect that
-        if(stream.isMuted())
-        {
-            if(stream.getType() === "audio")
-                this.onAudioMute(stream.getParticipantId(), true);
-            else
-                this.onVideoMute(stream.getParticipantId(), true);
-        }
+        // Make sure track's muted state is reflected
+        if(stream.getType() === "audio")
```

Comment:
As you can see on the diff the code was like that, before. I will push a separate commit for that